### DISCUSSION TITLE
How to protect static files?

### DISCUSSION THREAD

**Discussion by tisonkun** [vercel/next.js#44309] [Help]
See also https://github.com/auth0/nextjs-auth0/issues/977#event-8104931241

### Description

```ts
      const ignorePaths = [login, callback, unauthorized, '/_next', '/favicon.ico'];
      if (ignorePaths.some((p) => pathname.startsWith(p))) {
        return;
      }
```

Currently, request to `/_next` always bypass the auth process. This can leak content when the hack figure out the path of the generated js file.

**Even if I force redirect `_next` routes it seems `/_next/static/chunks/pages/index-045929068d457202.js` still bypass the middleware.**

### Reproduction

https://everlasting-lgzkwcx3s-korandoru.vercel.app/_next/static/chunks/pages/index-045929068d457202.js

This page uses the basic middleware config:

```ts
import {withMiddlewareAuthRequired} from "@auth0/nextjs-auth0/edge";

export default withMiddlewareAuthRequired();
```

You can find the page content leaks by searching "Get started by editing" which is part of the content.

### SDK version

2.0.1

### Next.js version

13.1.0

### Node.js version

18

**Comment by jasminezhan** [comment]
你好，小红书合作欢迎加我微信：jasmine920930 备注小红书合作
